definitions.yml

```diff
@@ -0,0 +1,6 @@
+:sles12:
+  :yast_url: "http://download.suse.de/ibs/Devel:/YaST:/Head/SLE-12-SP1/"
+  :iso_url: "http://dist.suse.de/install/SLE-12-Server-GM/SLE-12-Server-DVD-x86_64-GM-DVD1.iso"
+:sles12-sp1:
+  :yast_url: "http://download.suse.de/ibs/Devel:/YaST:/Head/SLE-12-SP1/"
```

Comment:
yes, plan is to replace head with alwyas target project name, so for SP2 it will be probably Devel:YaST:SLE-12-SP2 instead of Devel:/YaST:/Head